### PR TITLE
Greedy method

### PR DESCRIPTION
**Pull request by lihach54nsk**
added greedy method

### CODE REVIEW

File: Commivoyajer_Framework/Methods/GreedyMethod.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                    resultIndex = index;
+                    resultLength = variant.JourneyLength;
+                }
+
+                if (resultLength > variant.JourneyLength)
+                {
+                    resultIndex = index;
+                    resultLength = variant.JourneyLength;
+                }
+                index++;
```
Comment (SmallBossGames) [inline Commivoyajer_Framework/Methods/GreedyMethod.cs:94]: You can use for loop instead

File: Commivoyajer_User_Interface/Form1.cs [hunk 2]
```diff
@@ -132,20 +135,33 @@ private void greedyMethodButton_Click(object sender, EventArgs e)
...
+
+            result.CalculationTime = stopWatch.ElapsedMilliseconds;
+            result.JourneyLength = _prepareData.CalculateJourneyLength(input, result.Sequence);
+
+            ShowDataInUI(coords, result);
         }
 
         private void ShowDataInUI(List<Input> coords, Output output)
         {
+            var sequenceString = "";
```
Comment (SmallBossGames) [inline Commivoyajer_User_Interface/Form1.cs:152]: Use StringBuilder here. It's not a TypeScript

File: Commivoyajer_Framework/Methods/GreedyMethod.cs [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                {
+                    distance = row[i];
+                    resultIndex = i;
+                }
+
+                if (distance > row[i])
+                {
+                    distance = row[i];
+                    resultIndex = i;
+                }
```
Comment (SmallBossGames) [inline Commivoyajer_Framework/Methods/GreedyMethod.cs]: Bodies of these operators are the same. And don't use equals for floating-point values

Comment (lihach54nsk) [reply Commivoyajer_Framework/Methods/GreedyMethod.cs]: fixed

File: Commivoyajer_Framework/Methods/GreedyMethod.cs (hunk 1 above)
Comment (lihach54nsk) [reply Commivoyajer_Framework/Methods/GreedyMethod.cs:94]: lazy

File: Commivoyajer_User_Interface/Form1.cs (hunk 2 above)
Comment (lihach54nsk) [reply Commivoyajer_User_Interface/Form1.cs:152]: lazy

**Pull request merged**